src/Data/Swagger/Optics.hs

```diff
@@ -0,0 +1,560 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+-- |
+-- Module:      Data.Swagger.Optics
+-- Maintainer:  Andrzej Rybczak <andrzej@rybczak.net>
+-- Stability:   experimental
+--
+-- Lenses and prisms as labels for Swagger.
+--
+-- You can use them with the optics library, for example:
+--
+-- @
+-- data T = ...
```

Comment:
Add `{-# LANGUAGE OverloadedLabels #-}`

IIRC you need to do `\{-# LANGUAGE` so haddock isn't confused